ee/clickhouse/queries/person_distinct_id_query.py

```diff
@@ -21,6 +21,21 @@ def get_team_distinct_ids_query(team_id: int) -> str:
         return substitute_params(GET_TEAM_PERSON_DISTINCT_IDS, {"team_id": team_id})
 
 
+def get_team_distinct_ids_query_without_substitution() -> str:
```

Comment:
Q: Why not make the team_id argument optional to the other function? We're doubling our surface area here for APIs we need to maintain.

Reply:
Yep, absolutely. And at least as bad if not worse... the queries worked but only because magically and unrelatedly code elsewhere was substituting team_id in.

When I flipped to start passing team_id along, I realised it was called from a lot of places I'd not have time to properly check them all _and_ do support hero. 

@liyiy has kindly agreed to pick it up... which may well mean going for a completely different solution even :)

Reply:
i'm happy to pick up where this PR left off if you want me to though it would make more sense if you continued this since you have the most context and I'll just take over a couple of support hero questions xD